game/scripts/npc/abilities/clinkz_burning_army.txt

```diff
@@ -22,7 +22,7 @@
 
     // Time
     //-------------------------------------------------------------------------------------------------------------
-    "AbilityCooldown"                                     "110"
+    "AbilityCooldown"                                     "90" //OAA
```

Comment:
Clinkz Army CD's could probably be further reduced to a 90/80/70/60/50 as I don't think it has much impact outside of duel and I feel like the further cd buff would allow it to be used for bosses.